etcd/Dockerfile

```diff
@@ -22,6 +22,13 @@ COPY uninstall.sh /usr/bin/uninstall.sh
 
 COPY tmpfiles.template config.json.template service.template manifest.json /exports/
 
+RUN mkdir -p /exports/hostfs/usr/local/bin/ && cp /usr/bin/etcdctl /exports/hostfs/usr/local/bin/etcdctl
+
+# RUN mkdir -p /exports/hostfs/usr/local/bin/ && echo "runc exec -- etcd /usr/bin/etcdctl \$@" \
+# > /exports/hostfs/usr/local/bin/etcdctl && chmod +x /exports/hostfs/usr/local/bin/etcdctl
+
+RUN mkdir -p /exports/hostfs/etc/etcd && cp /etc/etcd/etcd.conf /exports/hostfs/etc/etcd
```

Comment:
I tried this, and this works, but what about the data dir? By default, each installation of the container will use the same host location for the data dir, unless the user changes it in the config file. The user would also have to change the ports in the config file to run two of these together, so maybe this isn't a problem?